### ISSUE TITLE
Add pwsh dictionary

### ISSUE THREAD

**Issue by AnderssonPeter** [AnderssonPeter/PowerType#3]

**Comment by Juanma7882** [comment]
Well I'm done taking my exams, now I'm going to work on adding some PowerShell commands.

**Comment by Juanma7882** [comment]
<img width="349" height="58" alt="Image" src="https://github.com/user-attachments/assets/3bd1ee6d-c9c4-46e6-b741-3f96afa7cdec" />

I have a question: How can I get it to enter console commands if it needs a key and empty strings aren't accepted? Can I configure it somewhere?
I have the changes in my fork currently not working because of that problem.

**Comment by AnderssonPeter** [comment]
Could you provide a example of what you want to write in the terminal and what it should suggest?

**Comment by Juanma7882** [comment]
Yes. For example, if you type "ip," it should suggest "ipconfig." Most console commands are typed directly without any preceding words.

<img width="959" height="509" alt="Image" src="https://github.com/user-attachments/assets/e2b4a4fd-a607-443e-b07e-ec1f5d79ac03" />

**Comment by AnderssonPeter** [comment]
Sadly powertype only works when you have completed the dictionaries keys. So I don't think it could complete your example.

Also the dictionary for pwsh should contain information about the parameters, commands and flags the pwsh.exe can take?

The example in your screenshot seems to be a list of common windows cli tools?

**Comment by Juanma7882** [comment]
If they are common Windows CLI tool commands, I didn't know they were different. When I searched, I saw more CLI commands than PowerShell commands, but I'm going to correct that.
These are pwsh commands, right?
Get-Help -Name Get-Command -Full
Get-Help -Name Get-Command -Detailed
Get-Help -Name Get-Command -Examples
Get-Help -Name Get-Command -Online
Get-Help -Name Get-Command -Parameter Noun
Get-Help -Name Get-Command -ShowWindow

**Comment by AnderssonPeter** [comment]
Sorry for the late response, this task is more about implementing a dictionary that descibes the command/parameters/flags that `pwsh.exe` accepts, if you write `pwsh -help` you will get the information.

**Comment by AnderssonPeter** [comment]
If you want to implement one for `ipconfig` that should be it's own dictionary.
